### ISSUE TITLE
Unusual sensor

### ISSUE THREAD

**Issue by btro1404** [abusenius/insaned#18]
Hey Alex,

I checked out a Canon "CanoScan LIDE 20" with insaned and it doesn't work together. I think it's because of the unusual name of the buttons that contains a space in their name:
```
$ insaned -L
List of sensors for device 'plustek:libusb:001:008':
    button 0    [no]
    button 1    [no]
    button 2    [no]
```
If I press a button, I expected a syslog entry of a missing script handler but there is no one.
Do you see a possibility to support these unusual names to use the scanner together with insaned?

Thank you, and
Gruß Bernd

**Comment by hannesrauhe** [comment]
Looking at this line:
`if (system((handler + " " + mCurrentDevice).c_str()) < 0) {`

I guess, insaned will execute a script named "button" and put 0,1,2 as first parameter. 
Running `./insaned --dont-fork -vv` (stop the service before), should give you some more information....

**Comment by btro1404** [comment]
No, you are wrong: A script named `button` isn't executed. And `insaned --dont-fork -vv` do not give a hint because it do not notice about a pressed button
